SDK Showcase/SDK Showcase/Model/Action.swift

```diff
@@ -5,4 +5,9 @@ import Foundation
 struct Action: AppModel {
     let name: String
     let description: String
+    var defaultValue: String = ""
+    var boolValue = false
+    var value: String = ""
```

Comment:
What is `defaultValue`, `boolValue` and just `value` mean? What does it "say" about the `Action`? Perhaps you wanted to have `ActionType.string(..)`, ActionType.bool(yes/no)` etc?

Reply:
`defaultValue` is - as the naming is suggesting - the default value for an action
boolvalue/value has been replaced by generic Binding.
enum type cannot process Binding thus I'm not able to do what you suggest.